### PR TITLE
Feat/cell

### PR DESCRIPTION
**Pull request by mel-langhoff**
Basic Cell class created, does not contain any active gameplay logic

### CODE REVIEW

File: spec/cell_spec.rb
Comment (kohljd) [inline spec/cell_spec.rb:1]: Rspec's "before(:each)" hook requires an "@" before variable names.
- cell -> @cell for line 5, 9, and 13

**Review by kohljd — CHANGES_REQUESTED** [review]

File: spec/cell_spec.rb
Comment (mel-langhoff) [reply spec/cell_spec.rb:1]: Added the "@"

Comment (kohljd) [reply spec/cell_spec.rb:1]: The variable name must be updated throughout the file in order for it to pass the tests.

```Ruby
RSpec.describe Cell do
    before :each do
        @cell = Cell.new
    end

    it "exists" do
        expect(@cell).to be_an_instance_of Cell
    end

    it "#empty?" do
        expect(@cell.token).to be nil
    end
end
```

File: spec/column_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require "./spec/spec_helper.rb"
+
+RSpec.describe Column do
+    before :each do
+        @column = Column.new
+    end
+
+    it "exists" do
+        expect(column).to be_an_instance_of Column
```
Comment (kohljd) [inline spec/column_spec.rb]: `expect(column) ` $\rightarrow$ `expect\@column)`

**Review by kohljd — CHANGES_REQUESTED** [review]

**Comment by mel-langhoff** [comment]
Changes should be completed in Column spec file

**Pull request merged**